fluss-server/src/main/java/org/apache/fluss/server/replica/ReplicaManager.java

```diff
@@ -514,6 +535,16 @@ public void putRecordsToKv(
                 timeoutMs, requiredAcks, entriesPerBucket.size(), kvPutResult, responseCallback);
     }
 
+    public void putRecordsToKv(
```

Comment:
Don't split this method into two just for testing purposes. If you feel the test changes are extensive, you can extract a common method within the test itself.